tests/request_methods/test_reports.py

```diff
@@ -160,10 +171,10 @@ def test_get_report_list(self):
         )
         self.assert_common_params(params)
         self.assertEqual(params['Action'], 'GetReportList')
-        self.assertEqual(params['Acknowledged'], acknowledged)
-        self.assertEqual(params['AvailableFromDate'], from_date.isoformat())
-        self.assertEqual(params['AvailableToDate'], to_date.isoformat())
-        self.assertEqual(params['MaxCount'], max_count)
+        self.assertEqual(params['Acknowledged'], str(acknowledged))
```

Comment:
Given other suggestions for converting a `bool` to either `"true"` or `"false"`, this and other parameters that use booleans would need a slight update to match (sorry for the nitpick :) ).